### ISSUE TITLE
Accessibility violation due to unlabeled visually hidden text area

### ISSUE THREAD

**Issue by flarnie** [andreypopp/react-textarea-autosize#251]
Running a Pa11y scan of a page which uses `react-textarea-autosize` triggers two WCAG2 AA violations:

```
      {
        "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Textarea.Name",
        "context": "<textarea style=\"min-height: 0px !important; max-height: none !important; height: 0px !important; visibility: hidden !important; overflow: hidden !important; position: absolute !important; z-index: -1000 !important; top: 0px !important; right: 0px !i...",
        "message": "This textarea element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.",
        "type": "error",
        "typeCode": 1,
        "selector": "...(omitted)"
      },
      {
        "code": "WCAG2AA.Principle1.Guideline1_3.1_3_1.F68",
        "context": "<textarea style=\"min-height: 0px !important; max-height: none !important; height: 0px !important; visibility: hidden !important; overflow: hidden !important; position: absolute !important; z-index: -1000 !important; top: 0px !important; right: 0px !i...",
        "message": "This form field should be labelled in some way. Use the label element (either with a \"for\" attribute or wrapped around the form field), or \"title\", \"aria-label\" or \"aria-labelledby\" attributes as appropriate.",
        "type": "error",
        "typeCode": 1,
        "selector": "...(omitted)"
      }
```

To fix it we just need to add the following changes [here](https://github.com/andreypopp/react-textarea-autosize/blob/master/src/calculateNodeHeight.js#L42): 
```
hiddenTextarea.setAttribute('tab-index', '-1');
hiddenTextarea.setAttribute('aria-hidden', 'true');
```
I will open a PR shortly.
Hope this is helpful!

**Comment by Andarist** [comment]
A PR would be much appreciated 🙏

**Comment by Andarist** [comment]
fixed in https://github.com/andreypopp/react-textarea-autosize/pull/252
